test/behavior/tendrl/cluster.feature

```diff
@@ -1 +1,91 @@
-Feature: CLUSTER
\ No newline at end of file
+Feature: CLUSTER
+
+  Scenario: Import a cluster
+    Given User is on "clusters" page
+     When User should see the "import" button
+      And User clicks on "import" button
+     Then User sees the "import-clusters" page
+
+  Scenario: Unmanage a cluster
```

Comment:
This scenario should come once the import is done so move it after `Scenario: Naming an imported cluster`